nengo_gui/namefinder.py

```diff
@@ -22,16 +22,17 @@ def find_names(self, net):
         classes = (nengo.Node, nengo.Ensemble, nengo.Network,
                    nengo.Connection)
 
-        for k in dir(net):
-            if not k.startswith('_') and k not in base_lists + all_lists:
-                v = getattr(net, k)
-                if isinstance(v, list):
-                    for i, obj in enumerate(v):
+        for class_attr in dir(net):
+            if(not class_attr.startswith('_')
+                and class_attr not in base_lists + all_lists):
```

Comment:
Ah!  Try to avoid this sort of indentation mismatch.  It makes it *very* hard to see that the second line of the if statement is part of the if statement condition rather than in the body....